File: app/src/main/java/com/example/sarada/moviereviews/models/Review.java
package com.example.sarada.moviereviews.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by sarada on 4/4/2018.
 */

public class Review {

    @SerializedName("author")
    private String author;
    @SerializedName("content")
    private String content;

    public Review(String author, String content) {
        this.author = author;
        this.content = content;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: app/src/main/java/com/example/sarada/moviereviews/models/Review.kt
package com.example.sarada.moviereviews.models

import com.google.gson.annotations.SerializedName

/**
 * Created by sarada on 4/4/2018.
 */
class Review(
    @field:SerializedName("author") var author: String, @field:SerializedName(
        "content"
    ) var content: String
)